docker-compose.yml

```diff
@@ -90,17 +61,16 @@ services:
     profiles:
       - node
     ports:
-      # the RPC endpoint - expose it on a different port to avoid conflicting with PXE
-      - 8999:8999/tcp
+      - 8080:8080/tcp
```

Comment:
Now we don't need to worry about the port collisions so I just used the port 8080 here.